packages/client/katello-pull-provider-migrate/katello-pull-provider-migrate.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/sh
+
```

Comment:
Maybe a good first check in this file would be to see if the system is registered? `subscription-manager identity` might handle that for ya